substrate/client/transaction-pool/Cargo.toml

```
@@ -51,6 +52,13 @@ sp-consensus = { workspace = true, default-features = true }
 substrate-test-runtime = { workspace = true }
 substrate-test-runtime-client = { workspace = true }
 substrate-test-runtime-transaction-pool = { workspace = true }
+thiserror = { workspace = true }
+tokio = { workspace = true, features = ["rt-multi-thread"] }
+tracing-subscriber = { workspace = true }
+#txtesttool = { version = "0.1.0", package = "substrate-txtesttool" }
+txtesttool = { git = "https://github.com/michalkucharczyk/tx-test-tool.git", package = "substrate-txtesttool" }
```

TODO after publishing substrate-txtesttool 0.2.0:

```suggestion
txtesttool = { version = "0.2.0", package = "substrate-txtesttool" }
```